cmd/tnf/fetch/fetch.go

```diff
@@ -0,0 +1,334 @@
+package fetch
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/test-network-function/cnf-certification-test/internal/registry"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	filterCsi              = "&filter=csv_description=iregex=CSI;organization==certified-operators"
```

Comment:
We should not filter based on CSI:
"&filter=csv_description=iregex=CSI;organization==certified-operators" => "&filter=organization==certified-operators"
CSI is only one type of operator (for storage)